### PR TITLE
Download needed packages and setup the project folders

### PR DESCRIPTION
**Pull request by laragurol**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,40 +1,62 @@
...
-    "dotenv": "^14.2.0",
-    "express": "^4.17.2",
-    "yarn-upgrade-all": "^0.6.1"
+    "bcrypt": "^5.0.1",
+    "body-parser": "^1.20.0",
+    "config": "^3.3.7",
+    "connect-mongo": "^4.6.0",
+    "cookie-parser": "^1.4.6",
+    "dotenv": "^16.0.1",
+    "ejs": "^3.1.8",
```
Comment (sarahmood) [inline package.json:24]: remove ejs package, since we are not building any views.

File: package.json [hunk 2]
```diff
@@ -1,40 +1,62 @@
...
+    "encrypt-cookie": "^1.1.5",
+    "error-handler": "^1.0.0",
+    "express": "^4.18.1",
+    "express-ejs-layouts": "^2.5.1",
```
Comment (sarahmood) [inline package.json:28]: remove ejs package, since we are not building any views.

**Review by sarahmood — COMMENTED** [review]
All agreed packages and folders were setup. Just make sure to remove couple of unused dependencies mentioned.

File: package.json [hunk 3]
```diff
@@ -1,40 +1,62 @@
...
   "scripts": {
-    "start": "nodemon src/app.js",
+    "start": "node src/app.js",
+    "devStart": "nodemon src/app.js",
     "test": "yarn validate && jest --passWithNoTests",
     "lint": "eslint --ignore-path .gitignore .",
     "format": "prettier --ignore-path .gitignore --write \"**/*.+(js|json)\"",
     "validate": "yarn format && yarn lint",
-    "validate-staged": "lint-staged",
-    "postinstall": "husky install && husky add .husky/pre-commit \"yarn validate-staged\" && husky add .husky/pre-push \"yarn test\""
```
Comment (halitbatur) [inline package.json:15]: why is this command removed this shouldn't be removed

**Review by halitbatur — CHANGES_REQUESTED** [review]
Looks good so far, fix the comments, merge it with the latest master branch and remove the __tests__ folder u created.

**Review by laragurol — COMMENTED** [review]
-fixed the "main" to src/app.js
-fixed env to initial state 
-reinstalled deleted husky dev 
-fixed deleted command on package.json
-uninstalled ejs and express-ejs-
-deleted tests folder
-deleted code in app file

**Comment by halitbatur** [comment]
@laragurol could u please do the following:
- Update your dev branch
- Create a new branch for this issue (delete the current branch)
- Download the needed packages only using `yarn` please dont use `npm` cause I see that there's package-lock.json
- and then do another PR

**Comment by halitbatur** [comment]
For now I will close this PR it has conflicting files, and it uses npm not yarn so it won't be merged.

**Pull request closed without merging**